tests/integration/fixtures/test_connect.py

```diff
@@ -1,3 +1,8 @@
 def test_databricks_connect(spark):
     rows = spark.sql("SELECT 1").collect()
     assert rows[0][0] == 1
+
+
+def test_databricks_connect_serverless(serverless_env, spark):
```

Comment:
Trigger linter with type hint

```suggestion
def test_databricks_connect_serverless(serverless_env, spark) -> None:
```